src/lib/phdi-building-blocks/phdi_building_blocks/standardize.py

```diff
@@ -1,93 +1,172 @@
 import phonenumbers
 import pycountry
 from typing import Callable, Literal, List
-
 from phdi_building_blocks.utils import find_patient_resources
+import copy
 
 
-def standardize_name(raw: str) -> str:
-    """trim spaces and force uppercase
-
-    >>> standardize_name(" JohN Doe ")
-    'JOHN DOE'
+def non_numeric_caps_standardization(raw_name: str) -> str:
     """
-
-    raw = [x for x in raw if not x.isnumeric()]
-    raw = "".join(raw)
-    raw = raw.upper()
-    raw = raw.strip()
-    return raw
+    Perform standardization on a provided name string. Name
+    standardization performs:
+    - removal of all numeric characters
+    - trimming of extra whitespace
+    - converting all characters to uppercase
+    For example, standardize_name(" JohN Doe ") is 'JOHN DOE'.
+    :param str raw: The raw string to standardize
+    :return: The standardized name string
+    :rtype: str
+    """
+    raw_name = [x for x in raw_name if not x.isnumeric()]
+    raw_name = "".join(raw_name)
+    raw_name = raw_name.upper()
+    raw_name = raw_name.strip()
+    return raw_name
 
 
-def standardize_patient_name(
-    bundle: dict, standardize: Callable = standardize_name
+def standardize_patient_names_in_bundle(
+    bundle: dict,
+    standardization_mode: str = "non_numeric_caps",
+    add_name_metrics: bool = True,
```

Comment:
Again, I'm _really_ opposed to including this in our production code.

Reply:
See comments regarding this functionality in the `geo.py` file. Removing here for the same reasons.